website/content/docs/discovery/dns.mdx

```diff
@@ -478,13 +478,20 @@ services from other namespaces or partitions the following form can be used:
 [tag.]<service>.service.<namespace>.ns.<partition>.ap.<datacenter>.dc.<domain>
 ```
 
-This is the canonical name of a Consul Enterprise service. Currently all parts must be
-present - in a future version (once the
+This is the canonical name of a Consul Enterprise service. Currently at least 2 of
+`[namespace, partition, datacenter]` must be present - in a future version (once the
 [`prefer_namespace` configuration](/docs/agent/config/config-files#dns_prefer_namespace) has been
```

Comment:
We should remove lines 483-486. The guidance on style is to only describe the current state of the product and avoid describing updates we will implement in the future.